### PR TITLE
Remove old SOAP-related code.

### LINKED ISSUE #17: Remove SOAP-related client code
We don't need it anymore, because we've recently switched to a new RESTful Sitewit API.

### PR DESCRIPTION
**Pull request by ekoval**
Closes   #17

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # [Changelog](https://github.com/yola/sitewit/releases)
 
+## 0.2.1
```
Comment (RayeN) [inline CHANGELOG.md]: Minor version bump, since we're introducing backward-incompatible changes.

File: requirements.txt [hunk 2]
```diff
@@ -4,7 +4,6 @@ flake8 < 3.0.0
 mock == 1.0.1
 nose < 2.0.0
 pinocchio < 1.0.0
-suds == 0.4
```
Comment (RayeN) [inline requirements.txt]: \o/

**Comment by RayeN** [comment]
Or you can skip version bump and changelog changes at all (I'll update changelog when introduce my changes)  We don't need to release new version right now.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by RayeN** [comment]
:+1:

**Comment by thomasw** [comment]
Yay!

**Pull request merged**